static/app/components/events/eventCause.tsx

```diff
@@ -30,26 +30,13 @@ function EventCause({group, eventId, project, commitRow: CommitRow}: Props) {
     projectSlug: project.slug,
   });
 
-  useEffectAfterFirstRender(() => {
-    if (fetching || !group?.id) {
-      return;
-    }
-
-    trackAdvancedAnalyticsEvent('issue_details.suspect_commits', {
-      organization,
-      count: committers.length,
-      project_id: parseInt(project.id as string, 10),
-      group_id: parseInt(group.id, 10),
-      issue_category: group?.issueCategory ?? IssueCategory.ERROR,
-    });
-  }, [
-    organization,
+  useRouteAnalyticsParams({
     fetching,
-    committers.length,
-    project.id,
-    group?.id,
-    group?.issueCategory,
-  ]);
+    count: committers.length,
+    project_id: parseInt(project.id as string, 10),
+    group_id: parseInt(group?.id || '', 10),
```

Comment:
@roggenkemper I don't think we even need a lot of these fields if the are already being set in static/app/views/organizationGroupDetails/groupDetails.tsx